src/frontend/sider/Fagsak/Personlinje/Behandlingsmeny/LeggTilEllerFjernBrevmottakere/useBrevmottakerSkjema.ts

```diff
@@ -153,9 +153,10 @@ export const useBrevmottakerSkjema = ({ eksisterendeMottakere }: Props) => {
             } else if (felt.verdi === '') {
                 return feil(felt, 'Feltet er påkrevd');
             }
-            return felt.verdi.length <= 10
+            //Sjekker at felter er 4 karakterer langt og er numerisk
+            return felt.verdi.length == 4 && /^\d{4}$/.test(felt.verdi)
```

Comment:
Litt usikker på om du trenger begge, men skader jo heller ikke 

```suggestion
            return /^\d{4}$/.test(felt.verdi)
```